requirements-base.in

```diff
@@ -0,0 +1,20 @@
+alpha-vantage~=2.3.1
```

Comment:
What is this .in file? I haven't heard of this before. Did this generated the requirements-base.txt?

Given that requirements.txt is (hopefully) improved, do you think we should still provide these files to the user? Curious to know your opinion

Reply:
`.in` files are transformed into `.txt` files by [`pip-compile` from pip-tools](https://github.com/jazzband/pip-tools/). This is one way to lock dependencies, but since Poetry has been introduced into the repo already [with its lock](https://github.com/DidierRLopes/GamestonkTerminal/blob/main/poetry.lock), I'll just yeet these for now :)